setup_scripts/moab_setup.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+ 
+#MOAB Installation
+INSTALL_ROUTE=$HOME
+cd $INSTALL_ROUTE
+
+apt install -y autoconf libtool make mpich libblas-dev liblapack-dev libhdf5-dev cmake
+
+mkdir MOAB
+cd MOAB
+mkdir bld
+mkdir install
+git clone https://bitbucket.org/fathomteam/moab
+cd moab
+git checkout Version5.0
+autoreconf -fi
+cd ../bld
+../moab/configure --prefix=$INSTALL_ROUTE/MOAB/install --with-hdf5=/usr/lib/x86_64-linux-gnu/hdf5/serial --enable-shared;
+make -j4
+make check
+make install
+printf 'export PATH=$PATH:%s/MOAB/install/bin' "$INSTALL_ROUTE">>$HOME/.bashrc
```

Comment:
You can just use `$INSTALL_ROUTE` (or `$INSTALL_ROOT`) in the string like you did for `$PATH`